scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -419,7 +419,8 @@ abstract class FixedPathSource(path: String*) extends FileSource {
   def localPaths = path.toList
   def hdfsPaths = path.toList
 
-  override def toString = getClass.getName + path
+  // The collection type of `path` causes equality issues in JobTest
+  override def toString = getClass.getName + path.mkString(",")
```

Comment:
Does the order of paths on this source matter?  If not, should we also sort this before concatenation?